ONSdigital/eq-author-prototypes
# Implement pulling selected route from state

This PR combines the route matching from react-router with the survey state in Redux:

- Flattened state out using Normalizr
- Route params (id of selected section/question/answer) matched against survey state

### How to test

Load a survey and check that clicking links in sidebar display correct title/label in the main view
